### PR TITLE
Circleci website build

### PR DESCRIPTION
**Pull request by alvin-huang**
This PR will move our current Packer builds for the website over to CircleCI!

This commit will need to be cherry-picked over into stable-website so it can build that branch.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    steps:
+      - checkout:
+          path: ~/project
+
+      # restores gem cache
+      - restore_cache:
+          key: static-site-gems-v1-{{ checksum "Gemfile.lock" }}
+
+      - run:
+          command: bundle check || bundle install --path vendor/bundle
```
Comment (dantoml) [inline .circleci/config.yml]: You may want to add `--retry=3` to `bundle install`, to account for random ruby gem installation flakiness

**Comment by dantoml** [comment]
Thanks for working on this!

**Comment by alvin-huang** [comment]
Great points @dantoml :D, touched up the file and repushed.

**Comment by preetapan** [comment]
Closing this PR since this was done against `stable-website` instead

**Pull request closed without merging**